app/components/member_list/index.js

```diff
@@ -40,12 +45,13 @@ export default class MemberList extends PureComponent {
         onListEndReached: PropTypes.func,
         onListEndReachedThreshold: PropTypes.number,
         sections: PropTypes.bool,
-        preferences: PropTypes.object
+        preferences: PropTypes.object,
+        loadingMembers: PropTypes.bool
     }
 
     static defaultProps = {
         onListEndReached: () => true,
-        onListEndThreshold: 10,
+        onListEndThreshold: 100,
```

Comment:
again I think this number is to high to set as a default prop, maybe just make sure to pass this prop to the component, I can see when other times this would be 50 or 60